.github/workflows/rebuild-llvm14.yml

```diff
@@ -175,6 +175,7 @@ jobs:
       shell: cmd
       run: |
         cd llvm
+        rmdir /s /q build-14.0
```

Comment:
Would removing of build directory be beneficial for older llvm versions?  Or we do not build them anymore?

Reply:
It doesn't matter as long as we don't run out of space. So I didn't do that for other builds.